management/src/main/java/org/kaazing/gateway/management/config/ServiceConfigurationBeanImpl.java

```diff
@@ -189,6 +189,26 @@ public String getAcceptOptions() {
                         jsonOptions.put("http.transport", httpTransport);
                     }
 
+                    Long tcpHandshakeTimeout = (Long) acceptOptions.remove("tcp.handshake.timeout");
+                    if (tcpHandshakeTimeout != null) {
+                        jsonOptions.put("tcp.handshake.timeout", tcpHandshakeTimeout);
+                    }
+
+                    Long sslHandshakeTimeout = (Long) acceptOptions.remove("ssl.handshake.timeout");
```

Comment:
I was thinking of removing the ssl changes for now, since I split the initial story in several stories, and there is a separate story for the ssl implementation.